app/views/IndexView.scala.html

```diff
@@ -69,70 +68,70 @@ <h2 class="govuk-heading-m">@messages("index.yourReturns")</h2>
 
 
 
-        @(periodInProgress, duePeriod, overduePeriods.size) match {
+        @(returnsViewModel.currentReturn, returnsViewModel.dueReturn, returnsViewModel.overdueReturns.size) match {
                 case (None, None, 0) => {}
-                case (None, None, 1) => {<p class="govuk-body">@messages("index.yourReturns.returnOverdue", overduePeriods.head.displayText)</p>}
+                case (None, None, 1) => {<p class="govuk-body">@messages("index.yourReturns.returnOverdue", returnsViewModel.overdueReturns.head.period.displayText)</p>}
                 case (None, None, x) => {<p class="govuk-body">@messages("index.yourReturns.onlyReturnsOverdue", x)</p>}
                 case (Some(inProgress), None, 1) => {
-                <p class="govuk-body">@messages("index.yourReturns.returnOverdue", overduePeriods.head.displayText)</p>
-                <p class="govuk-body">@messages("yourAccount.return.inProgress", inProgress.displayText)</p>
+                <p class="govuk-body">@messages("index.yourReturns.returnOverdue", returnsViewModel.overdueReturns.head.period.displayText)</p>
+                <p class="govuk-body">@messages("yourAccount.return.inProgress", inProgress.period.displayText)</p>
                 }
                 case (Some(inProgress), None, x) => {
                 <p class="govuk-body">@messages("index.yourReturns.onlyReturnsOverdue", x)</p>
-                <p class="govuk-body">@messages("yourAccount.return.inProgress", inProgress.displayText)</p>
+                <p class="govuk-body">@messages("yourAccount.return.inProgress", inProgress.period.displayText)</p>
                 }
 
                 case (None, Some(period), 0) => {
```

Comment:
Can we rename this variable to be return instead of period (and as with the other ones in this area)

Reply:
done